NewsAPI/NewsAPI/VC/TechnologiVC/TechnologyViewController.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  TechnologyViewController.swift
+//  NewsAPI
+//
+//  Created by Katerina on 05/12/2023.
+//
+
+import UIKit
+
+final class TechnologyViewController: BaseNewsVC {
+    
+    // MARK: - GUI Variables
+    private lazy var collectionView: UICollectionView = {
+        let layout = UICollectionViewFlowLayout()
+        
+        let width = (view.frame.width - 10)
+        layout.itemSize = CGSize(width: width, height: width/3)
+        
+        layout.minimumLineSpacing = 5
+        layout.minimumInteritemSpacing = 5
+        
+        let collectionView = UICollectionView(frame: CGRect(x: 0,
```

Comment:
will be enough 
`let collectionView = UICollectionView(frame: view.bounds, collectionViewLayout: layout)`